kooplearn/nn/functional.py

```diff
@@ -220,36 +236,38 @@ def eym_score_split(
         loss_on = (
             0.5
             * (
-                torch.trace(uc_cov_U1 @ uc_cov_U2)
+                torch.sum(uc_cov_U1 @ uc_cov_U2)
                 - torch.trace(uc_cov_U1)
                 - torch.trace(uc_cov_U2)
-                + torch.trace(uc_cov_V1 @ uc_cov_V2)
+                + torch.sum(uc_cov_V1 @ uc_cov_V2)
                 - torch.trace(uc_cov_V1)
                 - torch.trace(uc_cov_V2)
             )
             + d
         )
-        return score - metric_deformation * loss_on
+        return -1 * (score + metric_deformation * loss_on)
     else:
-        return score
-
+        return -1 * score
+    
 
 def eym_score(
     X: torch.Tensor,
     Y: torch.Tensor,
     mode: str = "split",
     metric_deformation: float = 1.0,
     center: bool = True,
+    cov: bool = True,
 ):
-    available_modes = ["split", "U_stat"]
+    available_modes = ["split", "Ustat"]
     if mode not in available_modes:
         raise ValueError(f"Unknown mode {mode}. Available modes are {available_modes}")
     if mode == "split":
-        return eym_score_split(X, Y, metric_deformation=metric_deformation)
-    else:
-        return eym_score_Ustat(
-            X, Y, metric_deformation=metric_deformation, center=center
-        )
+        if cov:
```

Comment:
What's the difference between `eym_score_split_cov` and `eym_score_split`? Have you tested both, and both are working?

Does it makes sense to keep them both? If not just keep the fastest of the two.